linAssgn1

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+clear
+echo -n "Enter the Process Id to be killed : " 
+read prid
+if [ $prid -le 1 -o $prid -ge 32768 ]
```

Comment:
I have found the maximum process id using "cat  /proc/sys/kernel/pid_max" and the value it returned is 32768.